video/video.ipynb

```diff
@@ -0,0 +1,4680 @@
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "074204d0-e2f9-4b28-aae2-0f0a10623096",
+   "metadata": {},
+   "source": [
+    "# <img src=\"https://raw.githubusercontent.com/iterative/datachain/main/docs/assets/datachain.svg\" style=\"height: 32px; width: 32px; vertical-align: bottom\"/> Tutorial: Working with Video Datasets\n",
+    "\n",
+    "This tutorial dives into techniques to manage video datasets datasets.\n",
+    "\n",
+    "**📋 Topics covered:**\n",
+    "1. Building a Video DataChain for [AVA Actions](https://research.google.com/ava/) video dataset\n",
+    "2. Creating a Data Model for YOLOv11 Pose Detection projects\n",
+    "3. Integrating Video-Level Annotations from CSV\n",
+    "5. Extract and Manage Video Frames\n",
+    "6. Running Pose Detection with YOLOv11 and Saving to DataChain\n",
+    "7. Visualizing Pose Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd887618-6f1b-4743-81cb-e97d0a157221",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "To start, install the dependencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "fa6fe04e-ead3-4c37-9df8-53cf7cf34ca0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip is available: \u001b[0m\u001b[31;49m24.3.1\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m25.0\u001b[0m\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install -q 'datachain[video]'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3acf791d-64bd-4c22-a7ea-40594bf75be5",
+   "metadata": {},
+   "source": [
+    "## Get data and create DataChain\n",
+    "\n",
+    "We will use **AVA Actions dataset** from [https://research.google.com/ava/download.html](https://research.google.com/ava/download.html#ava_actions_download). To download dataset, use the following script:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "4b252957-3b92-40f8-bbec-6cb9a358f1c3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Download '_-Z6wFjXtGQ.mkv'... OK\n",
+      "Download '_145Aa_xkuE.mp4'... OK\n",
+      "Download '_7oWZq_s_Sk.mkv'... OK\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "# Download video files list\n",
+    "DATA_DIR=\"data/ava\"\n",
+    "mkdir -p ${DATA_DIR}\n",
+    "\n",
+    "wget -q https://s3.amazonaws.com/ava-dataset/annotations/ava_file_names_trainval_v2.1.txt -O ${DATA_DIR}/ava_file_names_trainval_v2.1.txt\n",
+    "\n",
+    "# Download video files\n",
```

Comment:
[Q] Can we do this part with `from_storage`?

Reply:
even if we have to copy/use something from `gs://datachain-demo`

Reply:
Sure! 👍